modules/admin_manual/pages/configuration/server/security/oauth2.adoc

```diff
@@ -18,6 +18,7 @@ The OAuth 2.0 authorization framework enables a third-party application to obtai
 
 Here is an overview of how the process works:
 
+// Based on the kroki extension. Marked for ease of finding. Be aware that this renders by default to svg which has issues when building pdf. ATM, this is not an issue as this document is not part of the pdf (too deep in the level limit) and because the extension is not added to the pdf build call. This would need an adoption of the script. Be aware that tabs.js MUST be excluded in case (html only)! https://github.com/Mogztter/asciidoctor-kroki
```

Comment:
```suggestion
// Based on the kroki extension. Marked for ease of finding. Be aware that this renders by default to svg which has issues when building pdf. ATM, this is not an issue as this document is not part of the pdf (too deep in the level limit) and because the extension is not added to the pdf build call. This would need an adjustment of the script. Be aware that tabs.js MUST be excluded in case (html only)! https://github.com/Mogztter/asciidoctor-kroki
```

Reply:
Last sentence before the link: in case of what? Maybe: in case you want to try?